### DISCUSSION TITLE
Diff_ abundance methods

### DISCUSSION THREAD

**Discussion by patrickDsal** [ChiLiubio/microeco#132] [Ideas]
Hey,

as said a before: Great package! 
Is there any chance to include even more methods to assess differential abundance, especially for microbiome data.
Nearing et al., 2022 ( https://doi.org/10.1038/s41467-022-28034-z.) wrote a great paper comparing different methods.
It seems that ALDEx2 and Ancom-II produce the most robust results. Maybe one could implement those two methods in future updates of microeco !?

Best
Patrick

**Comment by ChiLiubio** [comment]
Hi Patrick,

Thanks very much for your suggestion! Yes. This paper is valuable for guiding users to select methods. I have read it once before, but the implementation is delayed. I will consider adding those in the next update of package.

Best,
Chi
